Sources/NIOTransportServices/NIOTSListenerBootstrap.swift

```diff
@@ -175,7 +175,7 @@ public final class NIOTSListenerBootstrap {
 
     /// Specifies a `ChannelOption` to be applied to the `NIOTSListenerChannel`.
     ///
-    /// - note: To specify options for the accepted `NIOTSConnectionChannels`s, look at `NIOTSListenerBootstrap.childChannelOption`.
+    /// > Note: To specify options for the accepted `NIOTSConnectionChannel`s, look at ``childChannelOption(_:value:)``.
```

Comment:
Just out of curiosity is `- Note` not working in Docc anymore?

Reply:
I dunno, they encourage use of this angle bracket version in the docs so I did that.